orbit/pkg/swiftdialog/utils.go

```diff
@@ -0,0 +1,43 @@
+package swiftdialog
+
+import (
+	"image"
+	"io"
+	"net/http"
+
+	// Packages below are not used explicitly, but are imported for initialization side-effects, which allows
+	// image.Decode to understand gif, jpeg, and png formatted images
+	_ "image/gif"
+	_ "image/jpeg"
+	_ "image/png"
+)
+
+const (
+	DefaultIconSize = uint(80)
+	WideIconSize    = uint(200)
+	WideAspectRatio = 1.8 // 9:5 aspect ratio
+)
+
+func GetIconSize(url string) (uint, error) {
+	resp, err := http.Get(url)
```

Comment:
As discussed on Slack, up to you but using a custom HTTP client would help ensure it has proper timeouts. Otherwise I think the gosec warning can be safely ignored - this will be an admin-provided URL or the default Fleet logo URL, not an random URL that any user can inject.